acceptance/tests/concatenated_option_flags.rb

```diff
@@ -0,0 +1,80 @@
+# This test is intended to ensure that command-line short options work
+# properly when concatenated.
+test_name "Facter short flags can be concatenated" do
```

Comment:
I would see this as an integration test in `spec_integration` where you could use `IntegrationHelper.exec_facter('-jp')` and expect not to output `/ERROR Facter::OptionsValidator - unrecognised option/`

Reply:
And something like `IntegrationHelper.exec_facter('-jpv')` to output  `/ERROR Facter::OptionsValidator - unrecognised option/`